src/s1reader/s1_reader.py

```diff
@@ -425,7 +425,8 @@ def _is_zip_annotation_xml(path: str, id_str: str) -> bool:
         return True
     return False
 
-def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str = 'vv'):
+def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str = 'vv',
```

Comment:
```suggestion
def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str='vv',
```

 I think within the brackets, spaces between operators can be removed